index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute for Cristiano Ronaldo</title>
+    <link rel="stylesheet" href="tribute.css">
+</head>
+<body>
+    <div id="main">
+        <h1 id="title">CR7 - The greatest of all time(GOAT)</h1>
+        <div id="img-div">
+            <a id="tribute-link" target="_blank" href="https://en.wikipedia.org/wiki/Cristiano_Ronaldo">
+            <img id="image" src="https://cdn.resfu.com/media/img_news/imagen-de-presentacion-de-cristiano-ronaldo-en-case-esports--twitter---case-esports.png?size=1000x&lossy=1&ext=jpeg"
+             width="50%" height="50%" alt="Photo of Ronaldo">
+            </a>
+             <figcaption id="img-caption" width="50%">Crstiano is the football legend because he shows the world to be the best.</figcaption>
+        </div>
+        <p id="tribute-info"><b>Cristiano Ronaldo dos Santos Aveiro:</b>(Portuguese pronunciation: [kɾiʃˈtjɐnu ʁɔˈnaɫdu]; born 5 February 1985) is a Portuguese professional footballer who plays as a forward for Premier League club Manchester United and captains the Portugal national team. 
+           <ul>
+                <b>Cr7 achievement:</b> <br>
```

Comment:
I suggest the <b> element Tag should be changed to an h2 -h6 Tag element but not a Bold element Tag

Reply:
- [ ] Good work using inline & polite clear comments, please remember to give a reason for your change request. 👍🏾